machine-learning/pyproject.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 packages = [{include = "app"}]
 
 [tool.poetry.dependencies]
-python = "=3.11.*"
+python = ">=3.10,<3.12"
```

Comment:
Do we need this constraint as well, or is just the renovate.json one enough?

Reply:
Renovate didn't like the asterisk. It changed the ML PR from chore to fix when I tried setting it to 3.11.*